### PR TITLE
Upgrade 1.30.x to 1.33.x

### PR DESCRIPTION
**Pull request by aman-agrawal**
Jira : https://devopsmx.atlassian.net/browse/OP-22085
Testing : Compile success, No new TCs failed bcoz of this change, deployed to ns=cvetarget aman1603/front50:16may1 - things LGTM
Note :
1. Will do the cleanup with separate PRs
2. Will modify github workflows & dockerfiles with separate commits - done (Now, part of this PR)
3. Add/change front50 code wrt final kork publish (if required)

### CODE REVIEW

**Comment by aman-agrawal** [comment]
I will not merge this PR to OES-1.30.1 (rather I will create new branch in opsmx/front50-oes)
To show the comparison of OSS-1.33.x & OES-1.30.1, this was done.

**Comment by meerajose** [comment]
Verified commits and .gradle file. 
All files are present. 
All the package versions are latest.
Please see the comments.

**Review by meerajose — CHANGES_REQUESTED** [review]

File: front50-test/front50-test.gradle [hunk 1]
```diff
@@ -15,15 +15,15 @@
...
 dependencies {
-  implementation("org.codehaus.groovy:groovy-all")
+  //api("org.apache.groovy:groovy:4.0.9")
 
   implementation project(":front50-api")
   implementation project(":front50-core")
   implementation project(":front50-s3")
-
-  implementation "org.codehaus.groovy:groovy"
+  testImplementation "org.apache.groovy:groovy:4.0.9"
```
[the remark this answers is not in the document]
Comment (aman-agrawal) [reply front50-test/front50-test.gradle:23]: https://github.com/OpsMx/front50-oes/blob/c2bdddc1bb25450a39e796ed0a95d445b937d40c/front50-test/front50-test.gradle#L23C3-L23C21

File: front50-test/front50-test.gradle [hunk 2]
```diff
@@ -15,15 +15,15 @@
  */
 
 dependencies {
-  implementation("org.codehaus.groovy:groovy-all")
```
[the remark this answers is not in the document]
Comment (aman-agrawal) [reply front50-test/front50-test.gradle:18]: https://github.com/OpsMx/front50-oes/blob/c2bdddc1bb25450a39e796ed0a95d445b937d40c/front50-test/front50-test.gradle#L23C3-L23C21

File: front50-test/front50-test.gradle (hunk 1 above)
[the remark this answers is not in the document]
Comment (meerajose) [reply front50-test/front50-test.gradle:23]: OK, Looks good to go

File: front50-test/front50-test.gradle (hunk 2 above)
[the remark this answers is not in the document]
Comment (meerajose) [reply front50-test/front50-test.gradle:18]: Found it is not present in 1.30.1 as well. good to go

File: front50-web/front50-web.gradle [hunk 3]
```diff
@@ -34,25 +35,41 @@ dependencies {
   implementation "io.spinnaker.kork:kork-config"
   implementation "io.spinnaker.kork:kork-web"
   implementation "io.spinnaker.kork:kork-exceptions"
-  implementation "com.squareup.retrofit:converter-jackson"
-  implementation "io.swagger:swagger-annotations"
```
[the remark this answers is not in the document]
Comment (meerajose) [reply front50-web/front50-web.gradle:38]: found it is not removed https://github.com/OpsMx/front50-oes/blob/OES-1.30.1/front50-web/front50-web.gradle
good to go

**Review by meerajose — APPROVED** [review]
Look good to merge

**Comment by j-sandy** [comment]
@aman-agrawal Could you resolve the conflicts so that it can be ready to merge ? Meanwhile I continue reviewing it.

**Comment by aman-agrawal** [comment]
> @aman-agrawal Could you resolve the conflicts so that it can be ready to merge ? Meanwhile I continue reviewing it.

https://github.com/OpsMx/front50-oes/pull/29#issuecomment-2116754586

**Comment by aman-agrawal** [comment]
> Look good to merge

https://github.com/OpsMx/front50-oes/pull/29#issuecomment-2116754586

**Comment by j-sandy** [comment]
So this PR is just for review, not to merge. I got it. It will be closed once reviewed.

File: front50-core/src/main/java/com/netflix/spinnaker/front50/model/plugins/PluginInfo.java [hunk 4]
```diff
@@ -28,7 +29,7 @@
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 import javax.annotation.Nonnull;
-import javax.validation.Valid;
+// import javax.validation.Valid;
```
Comment (j-sandy) [inline front50-core/src/main/java/com/netflix/spinnaker/front50/model/plugins/PluginInfo.java:32]: @aman-agrawal Remove the commented line from code.

File: front50-s3/src/test/groovy/com/netflix/spinnaker/front50/model/EventingS3ObjectKeyLoaderSpec.groovy [hunk 5]
```diff
@@ -19,7 +19,7 @@ package com.netflix.spinnaker.front50.model
 import com.fasterxml.jackson.databind.ObjectMapper
 import com.netflix.spectator.api.Registry
 import com.netflix.spinnaker.front50.config.S3MetadataStorageProperties
-import com.netflix.spinnaker.front50.config.S3Properties
+//import com.netflix.spinnaker.front50.config.S3Properties
```
Comment (j-sandy) [inline front50-s3/src/test/groovy/com/netflix/spinnaker/front50/model/EventingS3ObjectKeyLoaderSpec.groovy:22]: Clean up commented code.

File: front50-sql/src/test/kotlin/com/netflix/spinnaker/front50/model/SqlStorageServiceTests.kt [hunk 6]
```diff
@@ -294,8 +392,20 @@ internal object SqlStorageServiceTests : JUnit5Minutests {
     }
 
     afterAll {
-      jooq.close()
+     //jooq.close()
```
Comment (j-sandy) [inline front50-sql/src/test/kotlin/com/netflix/spinnaker/front50/model/SqlStorageServiceTests.kt:395]: Whether this block is required or not ? Plz verify, if not then clean it from here.

File: front50-test/front50-test.gradle [hunk 7]
```diff
@@ -15,15 +15,15 @@
...
 
   implementation project(":front50-api")
   implementation project(":front50-core")
   implementation project(":front50-s3")
-
-  implementation "org.codehaus.groovy:groovy"
+  testImplementation "org.apache.groovy:groovy:4.0.9"
   implementation "org.spockframework:spock-core"
   implementation "com.amazonaws:aws-java-sdk-s3"
+  //testImplementation "net.logstash.logback:logstash-logback-encoder:4.11"
```
Comment (j-sandy) [inline front50-test/front50-test.gradle:26]: Clean up the commented code.

File: front50-test/src/main/groovy/com/netflix/spinnaker/front50/utils/S3TestHelper.groovy [hunk 8]
```diff
@@ -56,7 +58,8 @@ class S3TestHelper {
    * jclouds.filesystem.basedir=/tmp/s3proxy
    *
    * $ ./s3proxy --properties s3proxy.properties
-   */
+   *//*
+
```
Comment (j-sandy) [inline front50-test/src/main/groovy/com/netflix/spinnaker/front50/utils/S3TestHelper.groovy:62]: If this segment of code is not required, please clean it up.

Comment (aman-agrawal) [reply front50-test/src/main/groovy/com/netflix/spinnaker/front50/utils/S3TestHelper.groovy]: https://github.com/OpsMx/front50-oes/pull/33

File: front50-test/front50-test.gradle (hunk 7 above)
Comment (aman-agrawal) [reply front50-test/front50-test.gradle]: Done : https://github.com/OpsMx/front50-oes/pull/33

File: front50-sql/src/test/kotlin/com/netflix/spinnaker/front50/model/SqlStorageServiceTests.kt (hunk 6 above)
Comment (aman-agrawal) [reply front50-sql/src/test/kotlin/com/netflix/spinnaker/front50/model/SqlStorageServiceTests.kt]: This block not required.
Cleanup Done : https://github.com/OpsMx/front50-oes/pull/33

File: front50-core/src/main/java/com/netflix/spinnaker/front50/model/plugins/PluginInfo.java (hunk 4 above)
Comment (aman-agrawal) [reply front50-core/src/main/java/com/netflix/spinnaker/front50/model/plugins/PluginInfo.java]: Done : https://github.com/OpsMx/front50-oes/pull/33

File: front50-s3/src/test/groovy/com/netflix/spinnaker/front50/model/EventingS3ObjectKeyLoaderSpec.groovy (hunk 5 above)
Comment (aman-agrawal) [reply front50-s3/src/test/groovy/com/netflix/spinnaker/front50/model/EventingS3ObjectKeyLoaderSpec.groovy]: Done : https://github.com/OpsMx/front50-oes/pull/33

**Review by j-sandy — APPROVED** [review]

**Pull request closed without merging**